kythe/java/com/google/devtools/kythe/util/KytheURI.java

```diff
@@ -147,6 +158,11 @@ public static String asString(VName vName) {
     return new KytheURI(vName).toString();
   }
 
+  /** Returns an equivalent Kythe ticket for the given {@link CorpusPath}. */
+  public static String asString(CorpusPath cp) {
```

Comment:
It's a little odd to have this here, but I also see the desire to not create a CorpusPaths class to host this one method.